services/cd-service/pkg/service/overview.go

```diff
@@ -477,6 +478,7 @@ func mapEnvironmentsToGroups(envs map[string]config.EnvironmentConfig) []*api.En
 		}
 	}
 	calculateEnvironmentPriorities(tmpEnvs)
+	sort.Sort(EnvironmentGroupsByDistance(result))
```

Comment:
if you sort here, remove line 469, it calls the same function.

I'm not sure what difference it would make, I think none. But I'm ok with doing the sorting here.

Reply:
What problem does it solve to do the sorting here? I'd like to investigate that a little bit. Sounds a bit like we have an unintended side effect here somewhere.

Reply:
please undo this change about sorting.

Reply:
Done!